web/src/components/dashboard/DashboardOnDesktop.tsx

```diff
@@ -0,0 +1,128 @@
+import { HomeBasedBusiness } from "@/components/data-fields/HomeBasedBusiness";
+import { FieldLabelDescriptionOnly } from "@/components/field-labels/FieldLabelDescriptionOnly";
+import { FilingsCalendar } from "@/components/filings-calendar/FilingsCalendar";
+import { Heading } from "@/components/njwds-extended/Heading";
+import { MediaQueries } from "@/lib/PageSizes";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { isHomeBasedBusinessApplicable } from "@/lib/domain-logic/isHomeBasedBusinessApplicable";
+import { QUERIES, routeShallowWithQuery } from "@/lib/domain-logic/routes";
+import {
+  Certification,
+  Funding,
+  OperateReference,
+  QuickActionLicenseReinstatement,
+  QuickActionLink,
+  QuickActionTask,
+  RoadmapDisplayContent,
+} from "@/lib/types/types";
+import { useMountEffectWhenDefined } from "@/lib/utils/helpers";
+import { LookupOperatingPhaseById } from "@businessnjgovnavigator/shared/";
+import { useMediaQuery } from "@mui/material";
+import { useRouter } from "next/router";
+import { ReactElement } from "react";
+
+interface Props {
+  quickActionTasks: QuickActionTask[];
+  quickActionLinks: QuickActionLink[];
+  quickActionLicenseReinstatements: QuickActionLicenseReinstatement[];
+  displayContent: RoadmapDisplayContent;
+  operateReferences: Record<string, OperateReference>;
+  fundings: Funding[];
+  certifications: Certification[];
+}
+
+export const DashboardOnDesktop = (props: Props): ReactElement => {
+  const { business, updateQueue } = useUserData();
+  const router = useRouter();
+  const operatingPhase = LookupOperatingPhaseById(business?.profileData.operatingPhase);
+  const isDesktopAndUp = useMediaQuery(MediaQueries.desktopAndUp);
+  const { Config } = useConfig();
+
+  useMountEffectWhenDefined(() => {
+    (async (): Promise<void> => {
+      if (isDesktopAndUp && business?.preferences.phaseNewlyChanged) {
+        if (!updateQueue || business?.onboardingFormProgress !== "COMPLETED") {
+          return;
+        }
+        await updateQueue.queuePreferences({ phaseNewlyChanged: false }).update();
+      }
+    })();
+  }, updateQueue);
+
+  const displayHomedBaseBusinessQuestion = (): boolean => {
+    if (!business) return false;
+    return (
+      isHomeBasedBusinessApplicable(business.profileData.industryId) &&
+      business.profileData.homeBasedBusiness === undefined &&
+      operatingPhase.displayHomeBasedPrompt
+    );
+  };
+
+  return (
+    <div className="display-none desktop:display-block" data-testid="desktop">
+      <RightSidebarPageLayout
+        mainContent={
+          <div className="margin-top-0 desktop:margin-top-0">
+            <UserDataErrorAlert />
+            <DashboardHeader />
+            <div className="margin-top-3">
+              {displayHomedBaseBusinessQuestion() && (
+                <div className="margin-bottom-4">
```

Comment:
[dust] I'd probably extract everything displayed as a result of the true conditional into its own component. This mirrors the behavior we see later with the `<QuickActionsContainer>`.